draft-lundberg-cose-two-party-signing-algs.md

```diff
@@ -360,6 +360,75 @@ and restricted for use with the ESP256 [I-D.jose-fully-spec-algs] signature algo
 
 # IANA Considerations {#IANA}
 
+## COSE Algorithms Registrations {#cose-alg-reg}
+
+This section registers the following values in the IANA "COSE Algorithms" registry [IANA.COSE]:
+
+- Name: ESP256-2p
+  - Value: TBD (Requested Assignment -65550)
```

Comment:
Values less than -65536 are [designated for private use](https://www.iana.org/assignments/cose/cose.xhtml#algorithms), which is why I've used values in that range as placeholders for prototypes. But I assume that for the actual IANA assignment, these should rather be in one of the ranges from -65536 to 65535?